### PR TITLE
fix(deps): Update deasync to 0.1.27

### PR DESCRIPTION
**Pull request by KaiPrince**
[US140756](https://rally1.rallydev.com/#/?detail=/userstory/641730463383&fdp=true): [awards-ui] Configure awards-ui repo with Dependabot and update dependencies

Related issue: https://github.com/Brightspace/frau-local-appresolver/issues/45

### CODE REVIEW

**Comment by KaiPrince** [comment]
I didn't include a lockfile because there wasn't already one present. But should there be?

**Comment by mbardakovD2L** [comment]
> I didn't include a lockfile because there wasn't already one present. But should there be?

I would keep it as-is, I _think_ whether or not there is a lockfile has to do with what version of node/npm this repo uses and updating that is probably out of scope for a single dependency

**Review by mbardakovD2L — APPROVED** [review]

**Comment by lloydwaddell** [comment]
this is consistent with previous dependency updates in the repo
https://github.com/Brightspace/frau-local-appresolver/commit/1cdef76801a6d97ce16c9a8f848ff95551ad93a7

**Comment by AntonBazhal** [comment]
TL;DR; do not add lockfile n this PR

Lockfiles do not matter much for libraries. npm only uses a project's lockfile and discards the ones that come from dependencies. So a lockfile here matters for dev experience only.

**Comment by bearfriend** [comment]
I'm confused about what this is effectively doing. `^0.1` will already match `0.1.27`. We try not to use more specific versions than are necessary.

**Comment by KaiPrince** [comment]
It is required by https://github.com/Brightspace/awards-ui/pull/90

**Comment by KaiPrince** [comment]
Closing because this is not necessary it turns out
